### ISSUE TITLE
Prepare satellite admin server for the new API implementation

### ISSUE THREAD

**Issue by ifraixedes** [storj/storj#6422]
## Motivation / Why?

We want to release the new satellite admin web app progressively while running the current one until the new one is complete and it offers all the functionalities that the current one has without issues.

## What needs to be done?

Make the appropriate changes in the current satellite admin server to be able to implement the authorization and new endpoints.

## Acceptance criteria

- Current satellite admin server is refactored to accept the authorization and endpoint implementation of the new satellite admin API.

**Comment by ifraixedes** [comment]
Commit to review https://review.dev.storj.io/c/storj/storj/+/11556

**Comment by ifraixedes** [comment]
I had to work a little bit more on this and fix the API generator because it was producing code that made our linter configuration yell.
I realized it because @jewharton had a commit that added the linter for the new admin web app UI source to our CI pipeline. His commit got merged before the one posted above.

The commit that fixes the API generator to address the linter issues is https://review.dev.storj.io/c/storj/storj/+/11655
